### ISSUE TITLE
Add simple tests for a multi-node cluster of badgerauth

### ISSUE THREAD

**Issue by amwolff** [storj/gateway-mt#162]
Depends on #164 (probably also #165).

**Comment by shaupt131** [comment]
"Simple tests" are ultimately looking for consistency and correctness.  "Advanced" testing (another ticket) will be focused on benchmarking clusters, etc...

Examples of simple tests: Create an N node cluster and create X operations for each node applied in random order and verify the state/consistency.

**Comment by storjBuildBot** [comment]
Change https://review.dev.storj.io/c/storj/gateway-mt/+/7467 mentions this issue.
